bigtable/tests/filters_integration_test.cc

```diff
@@ -57,23 +54,6 @@ class FilterIntegrationTest : public ::testing::Test {
 
   std::unique_ptr<bigtable::Table> CreateTable(std::string const& table_id);
 
-  /**
-   * Return all the cells included in @p request.
-   *
-   * TODO(#32) remove this when Table::ReadRows() is a thing.
-   */
-  std::vector<bigtable::Cell> ReadRows(bigtable::Table& table,
-                                       btproto::ReadRowsRequest request);
-
-  /**
-   * Return all the cells in the row pointed by @p rowkey, applying @p filter.
-   *
-   * TODO(#29) remove this when Table::ReadRow() is a thing.
```

Comment:
This line was supposed to be removed when `Table::ReadRow()` becomes a thing, but looks like all mentions of `ReadRow()` have now become `ReadRows()` — was it intentional to remove all calls to `ReadRow()`? Are there any tests remaining that use `ReadRow()` (singular)?

Or are these tests the ones that are being added as part of PR #182?

Reply:
When I first implemented these integration tests I used a single table for all the tests.  In that context it made sense to use `ReadRow()` for tests that involved a single row, and `ReadRows()` for tests with more rows.

Later we changed the tests to use a table per test, in that context using `ReadRows()` everywhere is simpler.  I did not notice that until I came back to simplify the code and use `Table::ReadRows()`.  So yes, removing the use of `ReadRow()` is intentional.

We only have unit tests in #182, I expect we will implement integration tests for #76 in a separate PR, as requested in #76.

Reply:
SGTM, thanks for the clarification!